packages/randoml/randoml.0.1.0/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+synopsis: "Generating cryptographically-secure random numbers"
+description:
+  "A library relying on Rust rand to generate random numbers. The random numbers are cryptographically secure."
+maintainer: ["david@o1labs.org"]
+authors: ["David Wong"]
+license: "ISC"
+homepage: "https://github.com/mimoo/randoml"
+bug-reports: "https://github.com/mimoo/randoml/issues"
+depends: [
+  "bignum" {>= "v0.14.0"}
+  "dune" {>= "2.8" & >= "2.8.0"}
+  "ppx_inline_test" {with-test & >= "v0.14.1"}
```

Comment:
```suggestion
  "conf-rust-2018"
  "ppx_inline_test" {with-test & >= "v0.14.1"}
```
There is also `conf-rust` if you don't need the specific edition

Reply:
It will not be able to pull `rand` though, you need to also vendor that, for example this is how is done by https://gitlab.com/tezos/tezos-rust-libs/

Reply:
because I also named my rust wrapper "rand" I'm not sure which rand you're talking about :D

[my rand](https://github.com/mimoo/randoml/blob/main/lib/rand/dune#L24) should be fine if I understand correctly because the public name is `randoml.rand`

intuitively the rust dependencies should be fine as well as cargo will download them during build. But now I think I understand that since opam install runs in a sandbox this won't happen... OK I see